client-rpc/src/rpc/sync_rpc.rs

```diff
@@ -1,22 +1,27 @@
 use jsonrpc_core::Result;
 use jsonrpc_derive::rpc;
 
+use crate::rpc::websocket_rpc::AddWalletCommand;
+use crate::server::{to_rpc_error, WalletRequest};
 use chain_core::state::account::StakedStateAddress;
 use client_common::tendermint::Client;
 use client_common::{Error, ErrorKind, PrivateKey, PublicKey, Storage};
 use client_core::{MultiSigWalletClient, WalletClient};
 use client_index::synchronizer::ManualSynchronizer;
 use client_index::BlockHandler;
-
-use crate::server::{to_rpc_error, WalletRequest};
-
+use serde_json::json;
+use std::sync::Mutex;
+use websocket::OwnedMessage;
 #[rpc]
 pub trait SyncRpc: Send + Sync {
     #[rpc(name = "sync")]
     fn sync(&self, request: WalletRequest) -> Result<()>;
 
     #[rpc(name = "sync_all")]
     fn sync_all(&self, request: WalletRequest) -> Result<()>;
+
+    #[rpc(name = "sync_continuous")]
```

Comment:
Perhaps `sync_addWallet` will be more descriptive of what it is doing?

Reply:
ok